benchmark/parse_output_gi.py

```diff
@@ -0,0 +1,79 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2015--, The WGS-HGT Development Team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+#
+# Parse output files of GI tools.
+#
+
+import sys
+import click
+from skbio import Sequence
+
+
+def parse_output_gi(genbank_fp,
+                    gi_fp):
+    """ Identify genes contained in GIs
+
+    Parameters
+    ----------
+    genbank_fp: string
+        file path to genome in GenBank format
+    gi_fp: string
+        file path to GI list
+
+    Returns
+    -------
+    output: string
+        gene names (protein_ids) separated by newline
+    """
+    genes = {}
+    gb = Sequence.read(genbank_fp, format='genbank')
+    for feature in gb.interval_metadata.features:
+        if feature['type_'] == 'CDS':
+            if 'protein_id' in feature:
+                protein_id = feature['protein_id'].replace('\"', '')
+                loc = gb.interval_metadata.features[feature]
+                start = loc[0][0] + 1
+                end = loc[0][1]
+                if protein_id not in genes:
+                    genes[protein_id] = [start, end]
+    genes_in_gi = {}
+    with open(gi_fp, 'r') as input_f:
+        for line in input_f:
+            l = line.strip().split()
+            if len(l) < 2:
```

Comment:
In my tests, the standard EGID output files always read `<start><\t><end><\n>`. Therefore `len(l)` always `== 2`. I am thinking that, if the user wants to add a comment in this file (e.g., `# GI no. 1<\n>`), or mistakenly added some new lines or whatever. This code will make the process safer.